src/DIRAC/WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -1474,10 +1476,10 @@ def getCurrentPID(self):
     #############################################################################
     def sendOutput(self, stdid, line):
         if stdid == 0 and self.stdout:
-            with open(self.stdout, "a+") as outputFile:
+            with open(self.stdout, "a+", encoding="utf-8") as outputFile:
```

Comment:
Do you want to handle the case of invalid unicode as well?
```suggestion
            with open(self.stdout, "a+", encoding="utf-8", errors="backslashreplace") as outputFile:
```